scripts/GetClrDbg.sh

```diff
@@ -76,97 +103,238 @@ generate_nuget_config()
     echo "</configuration>"                                                                                     >> NuGet.config
 }
 
-if [ -z "$1" ]; then
-    print_help
-    exit 1
-elif [ "$1" == "-h" ]; then
-    print_help
-    exit 1
-fi
+# Parses and populates the arguments
+parse_and_get_arguments()
+{
+    echo "Parsing Arguments..."
```

Comment:
Seems like the script shouldn't output this level of information except maybe in some sort of verbose mode.